packages/radix-ui-themes/src/components/alert-dialog.tsx

```diff
@@ -46,6 +50,7 @@ const AlertDialogContent = React.forwardRef<AlertDialogContentElement, AlertDial
               <div className="rt-BaseDialogScrollPadding rt-AlertDialogScrollPadding">
                 <AlertDialogPrimitive.Content
                   {...contentProps}
+                  asChild={false}
```

Comment:
Isn't this always going to override `asChild` from `contentProps`?
Also, I saw your note about panels like those needing `asChild`, what is the reasoning?

Reply:
> Isn't this always going to override `asChild` from `contentProps`?

This is a mistake, I'll fix. I went back and forth on these panels a bit (alert dialog content, dialog content, hover card content, popover content) and forgot to remove `asChild={false}` after the initial iteration.

My reasoning here is that the modal panel layouts might need to be quite flexible, and since they don't come with any fancy interactions or semantics, there's no good reason we'd prevent you from asChild'ing them into other elements.